### PR TITLE
Fix: Edge to edge on Sample app

### PR DESCRIPTION
**Pull request by AsimRibo**
## Summary

Added edge to edge support for sample app.

## Changes

Changes were simple, just adding `enableEdgeToEdge()` to Activities. Only exception is `JavaMainActivity` where you can't call this function. Since it is Java it should be possible to call it via `ClassName.method()` but I couldn't find which method that would be. So I added direct approach for edge to edge (you can see it in source of `enableEdgeToEdge()`).

I also removed `TopBar` to get full immersive experience of edge to edge.

### Type

- [ ] **Feature**: This pull request introduces a new feature.
- [x] **Bug fix**: This pull request fixes a bug.
- [ ] **Refactor**: This pull request refactors existing code.
- [ ] **Documentation**: This pull request updates documentation.
- [ ] **Other**: This pull request makes other changes.

#### Additional information

- [ ] This pull request introduces a **breaking change**.

#### Gesture navigation look

https://github.com/user-attachments/assets/4bb9a179-e6aa-480f-8516-cc7a47e92bb4

#### 3 button navigation look

https://github.com/user-attachments/assets/2a7a019f-788c-4fdc-a368-d9a18eb40b06

## Checklist

- [x] I have performed a self-review of my own code.
- [x] I have tested my changes, including edge cases.
- [ ] I have added necessary tests for the changes introduced (if applicable).
- [ ] I have updated the documentation to reflect my changes (if applicable).

## Additional notes

I tested this on light and dark mode on different Android versions. Looks good ✅ .

### CODE REVIEW

File: sample/src/main/kotlin/com/infinum/sentinel/sample/JavaMainActivity.java [hunk 1]
```diff
@@ -35,11 +40,34 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
...
+        if (isLightMode() && Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+            flags |= View.SYSTEM_UI_FLAG_LIGHT_STATUS_BAR;
+        }
+        decorView.setSystemUiVisibility(flags);
+    }
+
+    private boolean isLightMode() {
+        int currentNightMode = getResources().getConfiguration().uiMode & Configuration.UI_MODE_NIGHT_MASK;
+        return currentNightMode != Configuration.UI_MODE_NIGHT_YES;
+    }
```
Comment (KCeh) [inline sample/src/main/kotlin/com/infinum/sentinel/sample/JavaMainActivity.java:72]: 👀

**Review by KCeh — APPROVED** [review]
Good job 👏 
Just one question, there is no possibility that list will be stuck behind 3 button navigation? Ex. I can't scroll anymore and I can't click last element in the list because of 3 button navigation

**Comment by AsimRibo** [comment]
> Good job 👏 Just one question, there is no possibility that list will be stuck behind 3 button navigation? Ex. I can't scroll anymore and I can't click last element in the list because of 3 button navigation

It should be handled in the last commit: https://github.com/infinum/android-sentinel/pull/94/commits/f58b6f43dd9e08b159b52ebb1c739b608b60c793

Are you still having troubles with last element?

**Comment by KCeh** [comment]
> Are you still having troubles with last element?
> 
> But, in general, you should apply some sort if insets to last element on the screen. Similarly, with `Recyclers`, I think you can use `clipToPadding="false"` or something similar, there are guides on it.

All is good, I missed it 😅
